### ISSUE TITLE
1.13.* support

### ISSUE THREAD

**Issue by danielytics** [bitemyapp/revise#21]
@cesarbp, I would like to hear your thoughts and plans for 1.13.\* support.

I imagine its a pretty big job - new commands and possibly switch to the JSON protocol.

Have you thought about this yet? Any plans or possible timeline? Anything we can do to speed things up?

Thanks!

**Comment by cesarbp** [comment]
Hmm, yeah sorry, I left this project aside. I'll make time for it in these days. Any pull requests are welcome. In particular docs should be relatively easy to help with; some of the new functions in the changelog are yet to be documented. As for timelines, I don't really wanna promise anything, other than I'll start again working on it again. The hardest parts I think are the JSON stuff and the lambda issue.

**Comment by danielytics** [comment]
RE: the lambda issue, you mean #14?

I will take another look at the issue as soon as I have some time. I had started looking into it anyway and only stopped while waiting for you to fix the unit tests, but then got busy and never went back to it.

Do you think the JSON stuff might change this or is it basically a JSON translation of the existing protobuff AST?

**Comment by cesarbp** [comment]
Yes that lambda issue. I think the json stuff is basically just reimplementing the namespace that generates the protobufs. As for the lambda stuff, I was hoping for the macro to be able to number the variables properly without the need of some global variable. It should be not too hard using tools.analyzer or some such thing.

**Comment by danielytics** [comment]
I was thinking of modifying `compile-term` and numbering them there, probably using tools.analyzer or something as you say. I'll take a look at the weekend.

**Comment by cesarbp** [comment]
Yeah. I'd rather the whole thing was contained in the parser though.
